packages/esm-billing-app/src/types/index.ts

```diff
@@ -548,3 +548,19 @@ export type FacilityClaim = {
   externalId: string;
 };
 export type BillingPromptType = 'patient-chart' | 'billing-orders';
+
+export interface Schema {
+  services: {
+    all: Service[];
```

Comment:
```suggestion
    all: Array<Service>
```
its a nit but that the convention we are using on the project